File: Model/UserResult.cs
namespace SixDegrees.Model
{
    public class UserResult
    {
        internal string ID { get; set; }
        internal string Name { get; set; }
        internal string ScreenName { get; set; }
        internal string Location { get; set; }
        internal string Description { get; set; }
        internal long FollowerCount { get; set; }
        internal long FriendCount { get; set; }
        internal string CreatedAt { get; set; }
        internal string TimeZone { get; set; }
        internal bool GeoEnabled { get; set; }
        internal bool Verified { get; set; }
        internal long StatusCount { get; set; }
        internal string Lang { get; set; }
        internal string ProfileImage { get; set; }
    }
}

File: src/ClientApp/src/app/models/UserResult.ts
export interface UserResult {
    id: string;
    name: string;
    screenName: string;
    location: string;
    description: string;
    followerCount: number; // long
    friendCount: number; // long
    createdAt: string;
    timeZone: string;
    geoEnabled: boolean;
    verified: boolean;
    statusCount: number; // long
    lang: string;
    profileImage: string;
}

export class UserConnectionInfo {
    distance: number;
    connections: UserResult[];
}

export interface UserConnectionMap {
    [key: string]: UserConnectionInfo;
}
